File: src/main/java/io/kuzzle/sdk/responses/SecurityDocumentList.java
package io.kuzzle.sdk.responses;


import java.util.List;

import io.kuzzle.sdk.security.AbstractSecurityDocument;

public class SecurityDocumentList implements KuzzleList<AbstractSecurityDocument> {
  private List<AbstractSecurityDocument> documents;
  private long total;

  public SecurityDocumentList(List<AbstractSecurityDocument> roles, long total) {
    this.documents = roles;
    this.total = total;
  }

  public List<AbstractSecurityDocument> getDocuments() {
    return documents;
  }

  public long getTotal() {
    return total;
  }
}

File: src/main/java/io/kuzzle/sdk/responses/SecurityDocumentList.kt
package io.kuzzle.sdk.responses


import io.kuzzle.sdk.security.AbstractSecurityDocument
import io.kuzzle.sdk.util.Scroll

class SecurityDocumentList @JvmOverloads constructor(override val documents: List<AbstractSecurityDocument>, override val total: Long, val scroll: Scroll = Scroll()) : KuzzleList<AbstractSecurityDocument>
